src/groovy/org/pih/warehouse/api/StockTransferItem.groovy

```diff
@@ -79,7 +79,7 @@ class StockTransferItem {
         stockTransferItem.destinationBinLocation = productAvailability.binLocation
         stockTransferItem.quantity = productAvailability.quantityOnHand
         stockTransferItem.quantityOnHand = productAvailability.quantityOnHand
-        stockTransferItem.quantityNotPicked = productAvailability.quantityNotPicked > 0 ? productAvailability.quantityNotPicked : 0
+        stockTransferItem.quantityNotPicked = productAvailability.quantityNotPicked > 0 && productAvailability?.binLocation?.isPickable() ? productAvailability.quantityNotPicked : 0
```

Comment:
You should also check if item is recalled, 
add isPickable method to ProductAvailability, like it was done in AvailableItem

Reply:
Is it fine now?